.babelrc

```diff
@@ -1,4 +1,4 @@
 {
-  "presets": ["@babel/preset-env"],
+  "presets":[["@babel/preset-env", {"targets": "> 0.25%, not dead", "useBuiltIns": "usage", "corejs" : 3}]],
```

Comment:
We have browserlist field in `package.json` for that: https://github.com/denysdovhan/purifier-card/blob/master/package.json#L56-L59